en, zh: Add docs about passing raw TOML config (#765)
pingcap/docs-tidb-operator: en/configure-a-tidb-cluster.md

      cpu: 1
```

> Since TiDB Operator v1.1.6, support passing raw TOML config for the component:

Reviewer: ```suggestion
Since v1.1.6, TiDB Operator supports passing raw TOML configuration to the component:
```
@july2993 Please confirm whether this is correct.

      cpu: 2
```

> Since TiDB Operator v1.1.6, support passing raw TOML config for the component:

Reviewer: ```suggestion
Since v1.1.6, TiDB Operator supports passing raw TOML configuration to the component:
```

      enable-prevote: true
```

> Since TiDB Operator v1.1.6, support passing raw TOML config for the component:

Reviewer: ```suggestion
Since v1.1.6, TiDB Operator supports passing raw TOML configuration to the component:
```

```yaml
apiVersion: pingcap.com/v1alpha1
kind: TidbCluster
metadata:
  name: basic
spec:
.....
  pd:
    image: pingcap/pd:v4.0.6

>     config |

Reviewer 2: Please also update the `config:` in the en docs.

          level: information
```

> Since TiDB Operator v1.1.6, support passing raw TOML config for the component:

Reviewer: ```suggestion
Since v1.1.6, TiDB Operator supports passing raw TOML configuration to the component:
```
